sorc/rtma3d_read_diag.fd/makefile

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh -v
+
+# Define the fortran compiler and options
+
+#include ./Makefile.conf
+
+FC=ifort
+FLAGS=-convert big_endian 
```

Comment:
@MatthewMorris-NOAA 
I suggest to add a few options in FLAGS, as following:
FLAGS=-O3 -fp-model strict -free -assume byterecl -check bounds -g -traceback -implicitnone -qopenmp -convert big_endian
or at least
FLAGS=-check bounds  -g  -traceback  -implicitnone  -convert big_endian

Just a suggestion. The original one with only "-convert big_endian" is good.